lib/mutant/env/bootstrap.rb

```diff
@@ -106,6 +106,31 @@ def infect
         @integration = config.integration.new(config).setup
       end
 
+      # Return matched subjects
+      #
+      # @return [Enumerable<Subject>]
+      #
+      # @api private
+      #
+      def matched_subjects
+        Matcher::Compiler.call(self, config.matcher).to_a
```

Comment:
Not that it matters that much, but this collection is coerced to an Array. Is there a need for it to be an Array? If the object is properly enumerable (I assume it is and you didn't just define `#to_a` on it directly) can't that just be passed around?

If for some reason an Array is required, maybe the YARD doc should be updated to reflect that.